app/assets/javascripts/es6.js

```diff
@@ -0,0 +1,23 @@
+class Animal {
+  constructor(name) {
+    this.name = name;
+  }
+
+  speak() {
+    console.log(this.name + ' makes a noise.');
+  }
+}
+
+class Dog extends Animal {
+  speak() {
+    console.log(this.name + ' barks.');
```

Comment:
Expected indentation of 6 space characters but found 4.<br>Unexpected console statement.<br>Strings must use doublequote.